### ISSUE TITLE
Hyperparameters for unconditional 512 diffusion finetune

### ISSUE THREAD

**Issue by lopho** [EleutherAI/vqgan-clip#2]
I'm trying to reproduce the results of finetuning from the 512 conditional->unconditional ddpm model.
The only hyperparameters I could find so far, are the ones required for sampling, that are released with the notebooks. The rest is my best guess and all I could gather from the improved diffusion repo.

I would appreciate if you could release the parameters for lr, weight decay, batch size, etc.; as well as if the training was done on a subset/full set of ILSVRC20XX or ImageNet21K, and if augmentation was used.

@crowsonkb probably?

**Comment by crowsonkb** [comment]
I used for the most part the same hyperparameters as were used training the original model: batch size 256 (I used 8x A100 so I could do batch size 2 per GPU, so 16 per gradient accumulation step, so I did 16 gradient accumulation steps per optimizer step.) The learning rate was 1e-4 and I trained in fp16. The only hyperparameter I changed was the EMA decay which I changed from 0.9999 to 0.9995 because I only did 8100 steps of fine-tuning. I kicked in the EMA decay after some number of fine-tuning steps I forget (maybe 1000 or so). The training set was all of ILSVRC 2012, the same as the original training set except it wasn't using the labels. No augmentation was used.
